src/tutorials/annotation.md

```diff
@@ -130,34 +135,56 @@ and links to helpful guides and tutorials with the details.
+[BIDS annotation quickstart](https://www.hed-resources.org/en/latest/BidsAnnotationQuickstart.html).
 
 There are several online tools available at
-[https://hedtools.ucsd.edu/hed](https://hedtools.ucsd.edu/hed)
+[HED Tools Online](https://hedtools.org)
 to help you during this process:
 
 1. You can extract a ready-to-fill-in JSON sidecar template
 from a representative `events.tsv` file in your BIDS dataset.
 A step-by-step tutorial for doing this can be found in the
-[Create a JSON template tutorial](https://hed-examples.readthedocs.io/en/latest/BidsAnnotationQuickstart.html#create-a-json-template).
+[Create a JSON template tutorial](https://www.hed-resources.org/en/latest/BidsAnnotationQuickstart.html#create-a-json-template).
 
 2. Once you have a template, you can start editing the template directory,
 or you can convert the template to a spreadsheet and edit your
-annotations in Excel or other tool.
+annotations in Excel or another tool.
 Instructions for doing this are available in the
-[Spreadsheet templates tutorial](https://hed-examples.readthedocs.io/en/latest/BidsAnnotationQuickstart.html#spreadsheet-templates).
+[Spreadsheet templates tutorial](https://www.hed-resources.org/en/latest/BidsAnnotationQuickstart.html#spreadsheet-templates).
 
 This process and templates make it convenient to provide basic
-descriptions as well as HED tags for your dataset events.
+descriptions, as well as HED tags for your dataset events.
 
-A [HED annotation quickstart](https://hed-examples.readthedocs.io/en/latest/HedAnnotationQuickstart.html)
+A [HED annotation quickstart](https://www.hed-resources.org/en/latest/HedAnnotationQuickstart.html)
 outlines a step-by-step process for selecting HED tags during the annotation process.
+
+#### HED schemas
+
+The HED tags used to annotate data come from a controlled vocabulary called a HED schema.
+A HED schema is a structured vocabulary of terms consisting of top-level tags representing general categories in this vocabulary.
+Each top-level tag is the root of a tree containing tags falling into that category.
+This structure allows detailed and accurate annotation of events,
+machine validation of the annotations,
+and event description-based search across data collected in various studies.
+
+The rules for HED schema vocabularies and HED-compliant tools can be found in the [HED Specification](https://hed-specification.readthedocs.io/en/latest/).
+
+#### HED library schemas
+
+The HED standard schema contains basic terms that are common across most human neuroimaging, behavioral, and physiological experiments.
+The HED ecosystem schema libraries extend the standard HED schema with structured vocabularies, including terms unique to specific research fields.
+This allows the expansion of the HED vocabulary in a scalable manner to support specialized data annotations,
+for instance, electrophysiological events or language stimuli.
```

Comment:
Add links to the words of electrophysiological events (to HED-SCORE) and language stimuli (to LISA)

Reply:
Added [here](https://github.com/bids-standard/bids-starter-kit/pull/335/commits/fb1b7d87083388d5969afa7c10e59cca0b88214c)